lib/wallaby/selenium.ex

```diff
@@ -197,6 +197,11 @@ defmodule Wallaby.Selenium do
   @doc false
   defdelegate take_screenshot(session_or_element), to: WebdriverClient
 
+  @doc false
+  def take_fullpage_screenshot(session_or_element) do
+    WebdriverClient.take_fullpage_screenshot_moz(session_or_element)
```

Comment:
So, selenium is not only used for Firefox. You can use it with any browser/web driver it supports. I think this code won't work if you use selenium with chrome.